files/en-us/learn/performance/perceived_performance/index.html

```diff
@@ -88,7 +88,7 @@ <h3 id="interactive_elements_are_interactive">Interactive elements are interacti
 
 <h3 id="make_task_initiators_appear_more_interactive">Make task initiators appear more interactive</h3>
 
-<p>Making a content request on keydown rather than waiting for keyup can shave 200ms off the perceived load of the content. Adding an interesting but unobtrusive 200ms animation to that keyup even can reduce another 200ms of the perceived load. You're not saving 400ms of time, but the user doesn't feel like they're waiting for content until, well, until they're waiting for content.</p>
+<p>Making a content request on keydown rather than waiting for keyup can shave 200ms off the perceived load of the content. Adding an interesting but unobtrusive 200ms animation to that keyup event can reduce another 200ms of the perceived load. You're not saving 400ms of time, but the user doesn't feel like they're waiting for content until, well, until they're waiting for content.</p>
```

Comment:
As we are updating this line, I think we can make keydown, keyup inside `<code>` elements
```suggestion
<p>Making a content request on <code>keydown</code> rather than waiting for <code>keyup</code> can shave 200ms off the perceived load of the content. Adding an interesting but unobtrusive 200ms animation to that <code>keyup</code> event can reduce another 200ms of the perceived load. You're not saving 400ms of time, but the user doesn't feel like they're waiting for content until, well, until they're waiting for content.</p>
```

Reply:
Sorry about the bad PR name! Was made in a hurry.